CHANGELOG.md

```diff
@@ -35,6 +35,9 @@
         ([#2037](https://github.com/Automattic/pocket-casts-android/pull/2037))
     *   Sleep Timer restarts automatically if you play again within 5 minutes
         ([#2048](https://github.com/Automattic/pocket-casts-android/pull/2048))
+*   Updates:
+    *   Updated widget and added two new variants - small and large.
+        ([#2043](https://github.com/Automattic/pocket-casts-android/pull/2043))
```

Comment:
This isn't correct.

Reply:
We still need to align `version.properties` with the hotifx. Unless you plan to manage version code manually.